py/commands/update.py

```diff
@@ -0,0 +1,10 @@
+@bot.command()
+async def update(ctx):
+    data = 524288464422830095, 524288464422830095, 207188318130012160, 562086061153583122
```

Comment:
Could you make it so if the user has the Chaotic Dev role or the Chaotic Founder role instead of user IDs? Thanks man